public/app/features/search/components/ManageDashboards.tsx

```diff
@@ -153,7 +153,9 @@ const getStyles = stylesFactory((theme: GrafanaTheme) => {
       display: flex;
       flex-direction: column;
       flex: 1;
-      margin-top: ${theme.spacing.xl};
+      height: 100%;
+      padding-top: ${theme.spacing.lg};
+      margin-top: -${theme.spacing.md};
```

Comment:
It seemed the easiest to use padding instead of margin and compensate for the margin from `.page-action-bar`, which was pushing the search items down. Couldn't find a working combination using just height and we need `height: 100%;` rule to render the virtualised lists.

Reply:
ok...makes more sense. 👍